tests/unit/amo/components/TestSearchPage.js

```diff
@@ -48,44 +48,49 @@ describe('<SearchPage />', () => {
 
   it('renders a Paginate', () => {
     const root = render();
-    const paginator = findByTag(root, Paginate);
-    expect(paginator.props.count).toEqual(80);
-    expect(paginator.props.currentPage).toEqual(3);
-    expect(paginator.props.pathname).toEqual('/search/');
-    expect(paginator.props.queryParams).toEqual({ q: 'foo' });
+    const paginator = root.find(Paginate);
+    expect(paginator.prop('count')).toEqual(80);
```

Comment:
> I always hated having to switch between .equal(), .deepEqual(), and .strictEqual() in the past...

In general expect via jest means that `.equal()` and `deepEqual()` are replaced by `toEqual()` and `.strictEqual()` is replaced by `toBe()` so we could work on the basis of only using `toBe()` when we need the strict equality + identity check - leaving any potential perf issues aside from now - and then we can look at that separately if it matters.